main.go

```diff
@@ -0,0 +1,28 @@
+package main
```

Comment:
I guess the other question I have is why do you think this should only be an example?  This package (`server/`) is implementing a service that should compile to an executable.  The reusable components have already been packaged elsewhere.

Stated another way, why are we asking the ops team to implement main.go?  If anything, I would double down on this and move `-mkkey` and `-mkconfig` as flags here.